js/src/panes/pane.map.js

```diff
@@ -256,18 +256,19 @@ Wu.MapPane = Wu.Pane.extend({
     },
 
     _onMove : function () {
```

Comment:
Does this `_onMove` function ever fire?

Why is it necessary to comment out the event fire?